### PR TITLE
clio-40664 datepicker too many change events

### PR DESCRIPTION
**Pull request by JeffMarvin**

### CODE REVIEW

File: themis_components/thDatePicker/thDatePicker.directive.coffee [hunk 1]
```diff
@@ -9,11 +9,13 @@ angular.module("ThemisComponents")
       ngModel: "="
       dateFormat: "@"
       ngChange: "&"
+      ngBlur: '&'
```
Comment (seanhealy) [inline themis_components/thDatePicker/thDatePicker.directive.coffee]: We're not using this anymore.

File: themis_components/thDatePicker/thDatePicker.directive.coffee [hunk 2]
```diff
@@ -9,11 +9,13 @@ angular.module("ThemisComponents")
...
       condensed: "=?"
     bindToController: true
     controllerAs: "datepicker"
     controller: (PopoverManager, $element, $scope) ->
       tabEventCode = 9
+      updatingFromPopover = no
```
Comment (seanhealy) [inline themis_components/thDatePicker/thDatePicker.directive.coffee]: Perhaps toss a quick comment explaining what's up with this lock.

File: themis_components/thDatePicker/thDatePicker.directive.coffee (hunk 1 above)
Comment (JeffMarvin) [inline themis_components/thDatePicker/thDatePicker.directive.coffee]: Removed.

File: themis_components/thDatePicker/thDatePicker.directive.coffee (hunk 2 above)
Comment (JeffMarvin) [inline themis_components/thDatePicker/thDatePicker.directive.coffee]: Added. Please check that I am making sense on the new comments!

**Pull request merged**